### DISCUSSION TITLE
A short tutorial or code share on ONNX export and runtime

### DISCUSSION THREAD

**Discussion by harindercnvrg** [mindee/doctr#1155] [Q&A]
Can somebody please share end-to-end code to export detection and recognition model?
Once the export has been completed, how to load the models in ONNX runtime and run inference?

Thanks,

**Comment by felixdittrich92** [comment]
Hi @harindercnvrg 👋 ,

Thanks for asking. 
The onnx export is an experimential (not fully supported) feature currently. 
On the main branch you can export all models to onnx (up to the logits) but the loading via onnxruntime is still not integrated and on your own :)
